docs/blog/2020-05-07-gatsby-delivers-impossible-burgers-map/index.md

```diff
@@ -0,0 +1,104 @@
+tags:
+  - case-studies
+  - performance
+  - building-sites-faster
+---
+
+"Meat analogues" is a terrific punk band name. But it also describes one of the hottest current trends in food: plant-based protein products so realistic that they fool even hardcore carnivores, with a side of combatting climate change. One of the fastest growing alternative meat companies is Impossible Foods, particularly famous for their juicy yet plant-based Impossible Burger. When Burger King decided to offer a vegetarian version of their trademark Whopper, they chose the Impossible Burger---and Impossible Foods grew from supplying 100 restaurants to 5000, overnight.
+
+The company has been growing exponentially ever since, but unfortunately the original Impossible Foods website was not able to scale along with it. They turned to Gatsby to give their users a new and faster site that helped them solve their most pressing business problem: helping customers quickly find local retailers and restaurants offering Impossible Burgers. Gatsby, it turned out, was also the perfect platform for building an interactive store locator map that was beautiful, intuitive, and the fastest possible route for connecting users with the closest possible Impossible Burger.
+
+## The old website: Failing to find an Impossible Burger
+
+As demand skyrocketed, hungry customers flocked to the company's website to find out where they could experience an Impossible Burger in their local area. The original website showed users a static list of all Impossible Food locations, grouped by state.
+
+This meant that users had to scan a long and ever-growing list of retailers and restaurants, sorted alphabetically rather than geographically. They then had to use external tools like Google Maps to figure out the location closest by. Rather than navigate through this complicated process, many users simply gave up. Impossible Foods was losing an untold number of excited potential customers.
+
+"Our product would be in certain places, and people wouldn't know it was there," explains Ashley Geo, Head of Brand Marketing Communications for Impossible Foods. Alternatively, she adds, "It wouldn't be in other places, but people would guess that it was there" - only to leave empty-handed and disappointed.
+
+![](https://lh5.googleusercontent.com/UpLDOJ5IvE0tDhK9qqDSa6qbr9gWdM-wPRTZkI0P6XBDBI6gvKZk33qvcUVH88rFC_Ylh8GxTgnpvGZDPPuRMADalLJygiOzmlEY6SyawoFR3AH0BkhhLRSLwrB7aRMZXrCwkdJ6)
```

Comment:
why they are not in the blog directory 🤔

Reply:
created issue:

- #24626 `blog: Impossible food: Images should be inlined in repo and not hosted on googleusercontent.com`